update_copyright_hook

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
fairly sure that the portable solution is to have a space after the `#!` but not 100% sure...